docs/control_tuple_support.md

```diff
@@ -0,0 +1,207 @@
+# Custom Control Tuple (User Defined Control Tuple) Support
```

Comment:
"support" in the file name is redundant. How about renaming it to control_tuples.md